feeds/src/pages/Landing.js

```diff
@@ -1,50 +1,61 @@
 import React from 'react'
+import { useLocation } from 'react-router-dom'
 import { Listing } from 'components/listing'
 import { Header } from 'components/header'
 import { NodesLogos } from 'components/nodesLogos'
 import { SponsorsLogos } from 'components/sponsorsLogos'
 
-const LangingPage = () => (
-  <div className="page-wrapper landing-page">
-    <div className="page-container">
-      <Header />
-    </div>
-    <div className="head">
-      <div className="head__title">
-        <h1>
-          PRICE <br />
-          REFERENCE <br />
-          DATA{' '}
-        </h1>
-        <div className="square"></div>
+const COMPARE_OFFCHAIN = 'compare_offchain'
+
+function useQuery() {
+  return new URLSearchParams(useLocation().search)
+}
+
+const LangingPage = () => {
+  const query = useQuery()
+
+  return (
+    <div className="page-wrapper landing-page">
+      <div className="page-container">
+        <Header />
+      </div>
+      <div className="head">
+        <div className="head__title">
+          <h1>
+            PRICE <br />
+            REFERENCE <br />
+            DATA{' '}
+          </h1>
+          <div className="square"></div>
+        </div>
+      </div>
+      <div className="page-container">
+        <section>
+          <h3>Decentralized Oracle Networks for Price Reference Data</h3>
+          <p>
+            The Chainlink Network provides the largest collection of secure and
+            decentralized on-chain price reference data available. Composed of
+            security reviewed, sybil resistant and fully independent nodes which
+            are run by leading blockchain devops and security teams. Creating a
+            shared global resource which is sponsored by a growing list of top
+            DeFi Dapps.
+          </p>
+          <p>
+            Please feel free to look into the details of each Decentralized
+            Oracle Network listed below. You can easily use these oracle
+            networks to quickly and securely launch, add more capabilities to
+            and/or just greatly improve the security of your smart contracts.
+          </p>
+        </section>
+        <section>
+          <Listing compareOffchain={query.get(COMPARE_OFFCHAIN) === 'true'} />
```

Comment:
Doesn't look like we use query anywhere else, maybe our hook should be a little more specialized? `useOffchainQuery()`